pkgs/by-name/la/lazyjj/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  binutils-unwrapped,
+  fetchFromGitHub,
+  makeWrapper,
+  jujutsu,
+  rustPlatform,
+  testers,
+  lazyjj,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "lazyjj";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "Cretezy";
+    repo = "lazyjj";
+    rev = "v${version}";
+    hash = "sha256-VlGmOdF/XsrZ/9vQ14UuK96LIK8NIkPZk4G4mbS8brg=";
+  };
+
+  cargoHash = "sha256-TAq9FufGsNVsmqCE41REltYRSSLihWJwTMoj0bTxdFc=";
+
+  postInstall = ''
+    wrapProgram $out/bin/lazyjj \
+      --prefix PATH : ${
+        lib.makeBinPath [
+          binutils-unwrapped
+          jujutsu
+        ]
+      }
+  '';
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  nativeCheckInputs = [
+    binutils-unwrapped
+    jujutsu
+  ];
+
+  passthru.tests.version = testers.testVersion { package = lazyjj; };
+
+  meta = with lib; {
+    description = "TUI for Jujutsu/jj";
+    homepage = "https://github.com/Cretezy/lazyjj";
+    license = licenses.apache_2_0;
+    maintainers = with maintainers; [ colemickens ];
+    platform = platforms.unix;
```

Comment:
issue: `platforms`, not `platform`.

Also (sorry, that was my bad 😅), `buildRustPackage` sets the `platforms` automatically, so you can remove it here. 

```suggestion
```